specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/ServerGetWithVnet.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "testrg",
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "sku": {
+            "name": "Standard_D4s_v3",
+            "tier": "GeneralPurpose"
+        },
+        "properties": {
+            "fullyQualifiedDomainName": "c7d7483a8ceb.test-private-dns-zone.meru.com",
```

Comment:
Same incorrect domain name ` pgtestsvc4.postgres.database.azure.com` and in other examples